src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
+        <ul class="calendar__day"></ul>
```

Comment:
The `calendar__day` elements are empty `<ul>` elements. According to the task, they should contain `<li>` elements representing the days of the month. Make sure to add the appropriate `<li>` elements with the correct class names and data attributes if required.